### ISSUE TITLE
Crash when looking at Magical Wood from Extra Utilities 2

### ISSUE THREAD

**Issue by DestinyPlayer** [Darkhax-Minecraft/WAWLA#37]
http://pastebin.com/SVbLEMs4

When I hover over the Magical Wood block in JEI it crashes my whole game. Sorry for sparsity, but there's not really much I can decipher from that log.

**Comment by darkhax** [comment]
As far as I am aware, there is nothing I can do to solve this issue on my end. There has been a similar issue with Botania in the past, however that issue was fixed on Botania's side. I have created a new issue on Extra Utilities 2's GitHub which documents the issue a bit further and links to the solution implemented in Botania. We will have to wait for the Extra Utilities team to look into the issue before a proper solution can be implemented.

**Comment by darkhax** [comment]
So, I have worked out a solution to this issue, and it has been implemented. I am currently going through a transition in how I release updates, so this fix will not be publicly available for about a week. Once the new system is in place, new updates will be automatically published to CurseForge in real time, which will make things much easier for everyone involved.

**Comment by darkhax** [comment]
The new updates can be found [here](http://minecraft.curseforge.com/projects/wawla-what-are-we-looking-at/files/2287651). Has not been tested, however I am fairly confident it will work.
